### PR TITLE
review changes

### PR DESCRIPTION
**Pull request by srinidhis94**
**Changes**
* Adding boolean in postgres.
* Creating postgres example file.
* Support for precision and scale in floating point.

### CODE REVIEW

**Comment by srinidhis94** [comment]
@PumpkinSeed , @borosr  can you guys check this ? Have added few things missed from previous review and added boolean support which got missed earlier and  float precision support in action.

**Comment by srinidhis94** [comment]
CI/CD is broken. Shall I add postgres service and necessary changes or shall I remove postgres test in main_test.go ?

File: main_test.go [hunk 1]
```diff
@@ -67,6 +67,33 @@ func TestFuzz(t *testing.T) {
 	}
 }
 
+func TestFuzzPostgres(t *testing.T) {
```
Comment (PumpkinSeed) [inline main_test.go:70]: Let me first figure out how I can add postgres to the Github Actions.

File: drivers/postgres_example_test.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package drivers
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+)
+
+const CreateTable = `CREATE TABLE IF NOT EXISTS pg_data_types (
```
Comment (PumpkinSeed) [inline drivers/postgres_example_test.go:9]: Can you add some of them as NOT NULL, just to handle that case as well in the tests.

**Review by PumpkinSeed — CHANGES_REQUESTED** [review]
Amazing, thanks.

**Comment by PumpkinSeed** [comment]
I added the postgres service.

**Comment by srinidhis94** [comment]
There seems to be some problem with connecting to postgres. I don't think username and db are reflected correctly. Need to check this once

**Comment by borosr** [comment]
Hey, I guess the pipeline is failing because @PumpkinSeed added postgres to the actions, but you have to update your forked repo, to see it there. There is some help for that: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

**Comment by PumpkinSeed** [comment]
I will try to fix it because I guess it's something from our end. 

@srinidhis94 Let me have a bit of time to figure it out. Otherwise if you enjoy to contribute into the project feel free to contact me so we can have a more direct communication.

**Comment by srinidhis94** [comment]
Yea.. would like to contribute in free time. Any place to communicate as of now?

**Review by PumpkinSeed — APPROVED** [review]

**Pull request merged**